capsules/extra/src/nmea.rs

```diff
@@ -0,0 +1,155 @@
+// Licensed under the Apache License, Version 2.0 or the MIT License.
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+// Copyright Tock Contributors 2022.
+
+//! Provides userspace with access to NMEA sentences.
```

Comment:
I think this file is fine named as-is. This seems like a reasonable pass on what an NMEA Device interface should look like.